server/apiserver/apiserver.go

```diff
@@ -319,17 +360,31 @@ func (c *Client) Read(api *ApiServer) {
 		// 	}
 
 		case LoginUser:
-			fmt.Println("CreateUser message received.")
+			fmt.Println("LoginUser message received.")
 			var cu_msg LoginUserMsg
 			if err := json.Unmarshal(p, &cu_msg); err == nil {
 				log.WithFields(log.Fields{
 					"message": cu_msg,
 				}).Debug("received ws message")
-
-				timestamp := time.Now()
-				err := api.conService.SyncSetLastUserModification(cu_msg.Id, timestamp)
-				if err != nil {
-					// TODO(backend team): handle error response
+				userID := cu_msg.Email
+				//if userID == "" {
+				//	http.Error(w, errors.New("empty param: userID").Error(), http.StatusInternalServerError)
+				//	return
+				//}
+				byt = makeUserLoginResponseMsg(400, -1)
+				lastMove, getErr := api.conService.SyncGetLastUserModification(userID)
+				if getErr == consensus.NoSuchUser {
+					setErr := api.conService.SyncSetLastUserModification(userID, time.Unix(0,0)) // Default Timestamp for New Users
+					if setErr == nil {
+						byt = makeUserLoginResponseMsg(200, 0)
+					}
```

Comment:
What happens if setErr is not `nil`?

Reply:
ok we will just report another error then.